builder/busybox/busybox-1.33.1.config

```diff
@@ -0,0 +1,1208 @@
+#
```

Comment:
Can we go the fragment route on this file also? It would really reduce the code this repo is responsible to maintain and make it very clear what we are adding that pertains to our needs.

Reply:
No we can't. The upstream configs for nerves_initramfs don't have busybox enabled at all so there is no busybox config to fragment against: https://github.com/nerves-project/nerves_initramfs/blob/main/builder/configs/nerves_initramfs_arm_defconfig#L17

Reply:
Also, per Frank, nerves_initramfs is a dead project from his perspective and won't be maintained upstream. It just provides a convenient environment from which we can run `fwup -t complete` when standing up an Edge board for the first time